xcell/mappers/mapper_CatWISE.py

```diff
@@ -0,0 +1,117 @@
+from .mapper_base import MapperBase
+from .utils import get_map_from_points
+from astropy.table import Table
+import numpy as np
+import healpy as hp
+
+
+class MapperCatWISE(MapperBase):
+    def __init__(self, config):
+        """
+        config - dict
+          {'data_catalog': 'xcell/tests/data/
+          catwise_agns_masked_final_w1lt16p5_alpha.fits',
+           'mask': 'xcell/tests/data/MASKS_exclude_master_final.fits',
+           'mask_name': 'mask_CatWISE'}
+        """
+        self._get_defaults(config)
+        self.file_sourcemask = config.get('mask_sources', None)
+        self.ra_name = 'ra'
+        self.dec_name = 'dec'
+        self.radius_name = 'radius'
+        self.cat_data = None
+
+        self.npix = hp.nside2npix(self.nside)
+        # Angular mask
+        self.mask = None
+        self.delta_map = None
+        self.nl_coupled = None
+        self.dndz = None
+        # self.cat_redshift = None
+
+    # CatWISE catalog
+    def get_catalog(self):
+        if self.cat_data is None:
+            file_data = self.config['data_catalog']
+            self.cat_data = Table.read(file_data)
+            # Galactic coordinates
+            r = hp.Rotator(coord=['C', 'G'])
+            GLON, GLAT = r(self.cat_data['ra'], self.cat_data['dec'],
+                           lonlat=True)
+            self.cat_data['GLON'] = GLON
+            self.cat_data['GLAT'] = GLAT
+            # Angular and flux conditions
+            self.cat_data = self.cat_data[
+                (self.cat_data['w1'] <
+                 self.config.get('flux_max_W1', 16.4)) &
+                (np.fabs(self.cat_data['GLAT']) >
+                 self.config.get('GLAT_max_deg', 30))]
```

Comment:
The catalog seems to be already cut at `GLAT > 30`, so we don't need to include this

Reply:
Yes, this is true. The only condition necessary is the flux for W1